lib/server.js

```diff
@@ -36,8 +36,15 @@ function Server(originalCommand, baseDir, listenPort, controlPath) {
   // receive hooks etc.) are mounted on.
   this._baseApp = express();
 
-  this._serviceManager = new ServiceManager(this);
-  this._app = meshServer(this._serviceManager);
+  this._serviceManager = new ServiceManager(this, !!traceOptions.enableTracing);
+
+  var options = {};
+  if (!!traceOptions.enableTracing) {
+    options['trace.enable'] = true;
+    options['trace.db.path'] = this._baseDir;
+    options['trace.enableDebugServer'] = !!traceOptions.enableDebugServer;
```

Comment:
I suggest the only change necessary for a debug function would be here:
```
options['trace.enableDebugServer'] = !!process.env.STRONGLOOP_DEBUG_MINKELITE;
```

^--- above name is provisional, I'm not sure what the debug server is for.